app/lib/actions/candlepin/consumer/attach_subscriptions.rb

```diff
@@ -0,0 +1,25 @@
+module Actions
+  module Candlepin
+    module Consumer
+      class AttachSubscriptions < Candlepin::Abstract
+        input_format do
+          param :uuid, String
+          param :subscriptions, Hash
+        end
+
+        def plan(system, subscriptions)
+          plan_self(:uuid => system.uuid, :subscriptions => subscriptions)
+        end
+
+        def run
+          if input[:subscriptions]
```

Comment:
The lack of system.uuid would error out before here (like on line 11). I found that I would sometimes pass in no subscriptions, which isn't entirely useful but not necessarily worth an error.